source/dyaml/node.d

```diff
@@ -265,7 +265,7 @@ struct Node
         }
         /// Ditto.
         // Overload for types where we can make this nothrow.
-        this(T)(T value, const string tag = null) @trusted pure nothrow
+        this(T)(T value, const string tag = null) @trusted
```

Comment:
IIRC once one attribute is specified manually, the compile won't infer the rest.

Reply:
Double checked this just now - it doesn't appear to be the case.